src/components/Header/Header.jsx

```diff
@@ -14,14 +14,16 @@ const iconStyle = {
     color: '#69AD28',
 }
 
-export default function Header({ navItems }) {
+export default function Header({ }) {
     const [value, setValue] = useState(0)
     const [openPf, setOpenPf] = useState(false)
 
     const handleChange = (event, newValue) => {
         setValue(newValue)
     }
 
+    const nav = ['TRANG CHỦ', 'PHÁT HIỆN HÌNH ẢNH', 'BÀI VIẾT', 'THỰC VẬT', 'VỀ CHÚNG TÔI']
```

Comment:
Are you sure the navigation bar only has the text?

Reply:
i dont get it ?

Reply:
ok i got this